docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
+We currently do not recommend using this feature in production.
+</Admonition>
+
+Teleport's Workload Identity issues flexible short-lived identities intended
+for workloads. It is compatible with the industry-standard SPIFFE specification
```

Comment:
If we're not expecting readers to be familiar with SPIFFE, I would expand the abbreviation here and describe it in 1-2 sentences. The "What is SPIFFE" section can then describe SPIFFE in more detail.

Reply:
ae32679d56